test/e2e-techpreview/onclusterlayering_test.go

```diff
@@ -113,12 +116,14 @@ func TestOnClusterBuildsCustomPodBuilder(t *testing.T) {
 
 // Tests that an on-cluster build can be performed and that the resulting image
 // is rolled out to an opted-in node.
-func TestOnClusterBuildRollsOutImage(t *testing.T) {
+func TestOnClusterBuildRollsOutImageWithExtensionsInstalled(t *testing.T) {
 	imagePullspec := runOnClusterLayeringTest(t, onClusterLayeringTestOpts{
 		poolName: layeredMCPName,
 		customDockerfiles: map[string]string{
 			layeredMCPName: cowsayDockerfile,
 		},
+		useEtcPkiEntitlement: true,
```

Comment:
question: Is `useEtcPkiEntitlement` required for the extensions to work? I ask because the clusters that the CI system brings up don't have this enabled and consequently, it skips this test.

Reply:
It is not required, I have updated the test.